_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -286,6 +289,23 @@ function page_admin_scripts() {
 			'tracksUserData' => Jetpack_Tracks_Client::get_connected_user_tracks_identity(),
 			'currentIp' => function_exists( 'jetpack_protect_get_ip' ) ? jetpack_protect_get_ip() : false,
 			'lastPostUrl' => esc_url( $last_post ),
+			'theme' => array(
+				'name'      => $current_theme->get( 'Name' ),
+				'hasUpdate' => (bool) get_theme_update_available( $current_theme ),
+				'support'   => array(
+					'infiniteScroll' => current_theme_supports( 'infinite-scroll' ) || in_array(
+						$current_theme->get_stylesheet(), array(
+							'twentyten',
```

Comment:
Rather than hardcoding these theme names, I'd sooner see us do something like ...

```php
$themes = array();
foreach ( Jetpack::glob_php( JETPACK__PLUGIN_DIR . 'modules/infinite-scroll/themes/*.php' ) as $path ) {
    if ( is_readable( $path ) {
        $themes[] = basename( $path, '.php' );
    }
}
```

That way, when we add new themes support, we don't need to update it multiple places.